badge/src/main/java/com/decathlon/vitamin/compose/badge/VitaminCountBadges.kt

```diff
@@ -0,0 +1,256 @@
+            textStyle = VitaminTheme.typography.overline.copy(
+                color = VitaminTheme.colors.vtmnContentAccent,
+                fontWeight = FontWeight.W700,
+                fontSize = badgeSize.fontSize
+            )
+        ),
+        nbMaxChar: Int = 2,
+        isHiddenCountLeft: Boolean = false
+    ) {
+        VitaminCountBadge(
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            isHiddenCountLeft = isHiddenCountLeft,
+            badgeSize = badgeSize
+        )
+    }
+
+    @Composable
+    fun Default(
+        count: Int,
+        badgeSize: BadgeSize = BadgeSizeEnum.SMALL.badgeSize,
+        badgeStyle: BadgeStyle = BadgeStyle(
+            backgroundColor = VitaminTheme.colors.vtmnBackgroundAccent,
+            textStyle = VitaminTheme.typography.overline.copy(
+                color = VitaminTheme.colors.vtmnContentAccent,
+                fontWeight = FontWeight.W700,
+                fontSize = badgeSize.fontSize
+            )
+        ),
+        nbMaxChar: Int = 2,
+        isHiddenCountLeft: Boolean = false
+    ) {
+        VitaminCountBadge(
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            isHiddenCountLeft = isHiddenCountLeft,
+            badgeSize = badgeSize
+        )
+    }
+
+    @Composable
+    fun Reversed(
+        count: Int,
+        badgeSize: BadgeSize = BadgeSizeEnum.SMALL.badgeSize,
+        badgeStyle: BadgeStyle = BadgeStyle(
+            backgroundColor = VitaminTheme.colors.vtmnBackgroundAccent,
+            textStyle = VitaminTheme.typography.overline.copy(
+                color = VitaminTheme.colors.vtmnContentAccent,
+                fontWeight = FontWeight.W700,
+                fontSize = badgeSize.fontSize
+            )
+        ),
+        nbMaxChar: Int = 2,
+        isHiddenCountLeft: Boolean = false
+    ) {
+        VitaminCountBadge(
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            isHiddenCountLeft = isHiddenCountLeft,
+            badgeSize = badgeSize
+        )
+    }
+
+    @Composable
+    fun Alert(
+        count: Int,
+        badgeSize: BadgeSize = BadgeSizeEnum.SMALL.badgeSize,
+        badgeStyle: BadgeStyle = BadgeStyle(
+            backgroundColor = VitaminTheme.colors.vtmnBackgroundAccent,
+            textStyle = VitaminTheme.typography.overline.copy(
+                color = VitaminTheme.colors.vtmnContentAccent,
+                fontWeight = FontWeight.W700,
+                fontSize = badgeSize.fontSize
+            )
+        ),
+        nbMaxChar: Int = 2,
+        isHiddenCountLeft: Boolean = false
+    ) {
+        VitaminCountBadge(
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            isHiddenCountLeft = isHiddenCountLeft,
+            badgeSize = badgeSize
+        )
+    }
+
+}
```

Comment:
Same question. All these components have the same parameters (with the same default values) and the same implementation.